airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -1484,6 +1484,9 @@ definitions:
             examples:
               - "{{ record.id }}/{{ record.file_name }}/"
               - "{{ record.id }}_{{ record.file_name }}/"
+          $parameters:
```

Comment:
This was rather interesting. I was seeing a failing mypy run locally and on CI where it was complaining in the model to component factory that the `FileUploaderModel` didn't have a `parameters` field. And that lines up with our `declarative_component_schema.py` file not having it either. So I have no idea how CI was passing before this, but I added it in to fix it here.